src/compute/src/logging/persist.rs

```diff
@@ -42,8 +42,6 @@ pub(crate) fn persist_sink<G>(
         truncate,
     );
 
-    // We don't allow these dataflows to be dropped, so the tokens could
-    // be stored anywhere.
```

Comment:
Removed this comment, because it isn't strictly true: Introspection dataflows can be dropped as a reaction to a `DropInstance` command.